src/style/components/_items-list.scss

```diff
@@ -0,0 +1,95 @@
+.cs-items-list-container {
+  margin: 20px 0 0 0;
+}
+
+.cs-items-list-row {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 20px 0;
+}
+
+.cs-items-list-item {
+  //flex-grow: 1;
+  //display: inline-block;
+  width: 435px;
+  height: 300px;
+  border: solid 1px silver;
+  border-radius: 2px;
+  cursor: pointer;
+
+  &:hover {
+    //background-color: #e0e0e0;
+
+    .image {
+      .overlay {
+        opacity: 0.75;
+      }
+      img {
+        opacity: 1;
+      }
+    }
+  }
+
+  .image {
```

Comment:
Here and a bit down you're using plain classes like 'image', 'overlay'... We should respect here the BEM naming, establishing a new component name or continuing with it's current parent.

For example: cs-items-list-item-image, cs-items-list-item-image-overlay and so.